src/common/meta/src/cache/table/table_name.rs

```diff
@@ -0,0 +1,173 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use futures::future::BoxFuture;
+use moka::future::Cache;
+use snafu::OptionExt;
+use table::metadata::TableId;
+
+use crate::cache::{CacheContainer, Initializer};
+use crate::error;
+use crate::error::Result;
+use crate::instruction::CacheIdent;
+use crate::key::table_name::{TableNameKey, TableNameManager, TableNameManagerRef};
+use crate::kv_backend::KvBackendRef;
+use crate::table_name::TableName;
+
+/// [TableNameCache] caches the [TableName] to [TableId] mapping.
+pub type TableNameCache = CacheContainer<TableName, u32, CacheIdent>;
```

Comment:
Can we use the alias for `TableId` here?

Reply:
Good point, it should be `TableId`